tur/clanbomber/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://www.nongnu.org/clanbomber
+TERMUX_PKG_DESCRIPTION="The goal of ClanBomber is to blow away your opponents using bombs, but avoid being blown up yourself."
+TERMUX_PKG_LICENSE="GPL-3.0"
```

Comment:
This should be custom, as it contains `COPYING, LICENSE.DEJAVU`

Reply:
Fixed